### ISSUE TITLE
Focus ring is not visible on keyboard navigation

### ISSUE THREAD

**Issue by sbadau** [mozilla/blurts-server#1518]
[Affected versions]:
- Firefox 72.0.2

[Affected Platforms]:
- Windows 10 x64
- Windows 7 x64
- MacOS 10.13
- Ubuntu 18.04 x64

[Steps to reproduce]:
1. Navigate to https://monitor-v2.herokuapp.com/
2. Log into a Firefox Monitor account with breaches
3. On the main dashboard use your mouse to navigate through the breaches

[Expected result]:

Keyboard navigation is allowed and a focus ring should show the location on the page.

[Actual result]:

The focus ring is visible only on the buttons, it's not visible on links. 

[Additional Info]:

This issue is reproducible on all the browsers.

**Comment by EMMLynch** [comment]
Closing since we've redesigned the site and functionality since this was created. If you feel that this is still needed, please let me know.
